public/javascripts/common/UtilitiesPanomarker.js

```diff
@@ -268,8 +268,8 @@ function getCanvasCoordinate(origPov, newPov, canvasWidth, canvasHeight, iconWid
             outputCoord.x = offset.left;
             outputCoord.y = offset.top;
         } else {
-            outputCoord.x = null;
-            outputCoord.y = null;
+            outputCoord.x = -iconWidth;
```

Comment:
Is there a reason that this needed to be changed from `null` to `-iconWidth`? Couldn't your code just check if the values are null instead of checking if they are less than 0? I'm just concerned about the possibility that we don't know what the implications are of changing the output of this function!

Reply:
Were there other instances where this function was outputting negative values that were problematic?

Reply:
I think that this also means that the label will disappear before it's fully off screen, correct? I think that the labels should continue to show until they would fully be off screen!